ckcp/openshift_dev_setup.sh

```diff
@@ -211,18 +211,24 @@ install_cert_manager(){
 
 check_cert_manager() {
   # perform a dry-run create of a cert-manager
-  # Certificate resource in order to verify that CRDs are installed and all the
-  # required webhooks are reachable by the K8S API server.
+  # Certificate resource in order to verify that the cert-manager CRDs are installed on the Kubernetes API server
   local i=0
   until kubectl create -f "$CKCP_DIR/openshift/base/certs.yaml" --dry-run=client >/dev/null 2>&1; do
-    echo -n "."
+    printf "."
     sleep 5
     i=$((i+1))
     if [ "$i" == 12 ]; then
-      echo "[ERROR] Failed to deploy OpenShift Cert-Manager operator on the host cluster."
+      printf "[ERROR] The cert-manager CRDs are not yet installed on the Kubernetes API server.\n"
```

Comment:
The previous message was more clearly stating that it is a failure.
Proposing: "Failed to deploy the cert-manager operator. CRDs are not available.\n"